dbuf/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM {{ service_base_image.image }}
+
+COPY setup.py setup.cfg dbuf/
+COPY dbuf dbuf/dbuf
+RUN python3 -m pip install ./dbuf
+
+ENTRYPOINT ["python3", "-m", "dbuf"]
+CMD []
```

Comment:
This seems unnecessary.

Reply:
Prior images may have specified command arguments. It seems prudent to override them.